extensions/amp-script/0.1/test/integration/test-amp-script.js

```diff
@@ -86,10 +86,12 @@ describe
             .callsFake(() => false);
           browser.click('button#hello');
 
-          // Give mutations time to apply.
+          // Wait for mutations to be processed
           yield browser.wait(100);
-          yield poll('terminated', () => {
-            return element.classList.contains('i-amphtml-broken');
+
+          yield poll('mutations should be dropped', () => {
+            // Mutation was dropped, therefore no h1s were added.
+            return !doc.querySelector('h1');
```

Comment:
I think you can just `expect()` here without a poll. Polling for continued non-existence is unnecessary.